app/models/email_template.rb

```diff
@@ -42,6 +42,10 @@ def users
       User.mentee.where.not(registration_token: nil)
     when :unregistered_mentors
       User.mentor.where.not(registration_token: nil)
+    when :abandoned_mails_mentors
+      User.mentor.where(is_missing: true).where(missing_since: ((Time.now - 10.days)..Time.now))   
```

Comment:
what about people who haven't been reported as missing and they haven't used app for overall 25 days?